scripts/config.js

```diff
@@ -5,13 +5,113 @@
  */
 /* exported DevCommitmentConfig */
 /* exported ProdCommitmentConfig */
+/* exported CHL_BYPASS_SUPPORT */
+/* exported CHL_BYPASS_RESPONSE */
+/* exported PPConfigs */
 
-const DevCommitmentConfig = {
-    "G":"BIpWWWWFtDRODAHEzZlvjKyDwQAdh72mYKMAsGrtwsG7XmMxsy89gfiOFbX3RZ9Ik6jEYWyJB0TmnWNVeeZBt5Y=",
-    "H":"BKjGppSCZCsL08YlF4MJcml6YkCglMvr56WlUOFjn9hOKXNa0iB9t8OHXW7lARIfYO0CZE/t1SlPA1mXdi/Rcjo="
-}
+const CHL_BYPASS_SUPPORT  = "cf-chl-bypass"; // header from server to indicate that
+const CHL_BYPASS_RESPONSE = "cf-chl-bypass-resp"; // response header from server, e.g. with erorr code
 
-const ProdCommitmentConfig = {
-    "G":"BOidEuO9HSJsMZYE/Pfc5D+0ELn0bqhjEef2O0u+KAw3fPMHHXtVlEBvYjE5I/ONf9SyTFSkH3mLNHkS06Du6hQ=",
-    "H":"BHOPNAWXRi4r/NEptOiLOp8MSwcX0vHrVDRXv16Jnowc1eXXo5xFFKIOI6mUp8k9/eca5VY07dBhAe8QfR/FSRY="
+const exampleConfig = {
```

Comment:
This exampleConfig is super useful for a first-time user (it helped me familiarize myself with integrations)